pennylane/transforms/optimization/single_qubit_fusion.py

```diff
@@ -36,7 +36,7 @@ def single_qubit_fusion(
     (on the same qubit) with that property defined will be fused into one ``Rot``.
 
     Args:
-        tape (QNode or QuantumTape or Callable): A quantum circuit.
+        tape (QNode or QuantumTape or callable): A quantum circuit.
```

Comment:
There seem to be more `Callable` than `callable` in the code base. Is one correct the other not? Are they just different?

Reply:
Ah, I actually don't know, this is a bit of a leftover :D I'll just revert to stick to the majority vote.